rasa_nlu/data_router.py

```diff
@@ -308,18 +311,38 @@ def start_train_process(self,
         def training_callback(model_path):
             model_dir = os.path.basename(os.path.normpath(model_path))
             self.project_store[project].update(model_dir)
+            self._current_training_processes = \
```

Comment:
I guess `self._current_training_processes -= 1` should do the same?

Reply:
Sure